website/i18n/en/docusaurus-plugin-content-docs/current/routes/journal.md

```diff
@@ -0,0 +1,530 @@
+import RouteEn from '@site/src/components/RouteEn';
```

Comment:
EN docs and CN docs will be merged into one. The primary language of `website/docs/routes/journal.md` will be English unless the site provides Chinese only.